src/styles/blocks/cards/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $card-bg-color;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;

```

Comment:
This style should be added for all elements using `*` selector as it's the main rule how you calculate the size for your blocks through the page